tools/uniprotxml_downloader/uniprotxml_downloader.xml

```diff
@@ -1,5 +1,8 @@
-<tool id="uniprotxml_downloader" name="UniProt" version="2.2.0" profile="21.01">
+<tool id="uniprotxml_downloader" name="UniProt" version="2.3.0" profile="21.01">
     <description>download proteome as XML or fasta</description>
+    <macros>
+        <import>macros.xml</import>
```

Comment:
small macros you could include directly here

Reply:
Thanks, I considered it but arbitrarily chose macros.xml.